test/source/tests/util.ts

```diff
@@ -0,0 +1,33 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
```

Comment:
I guess these methods should go to `test/source/tests/page-recipe/compose-page-recipe.ts`